### PR TITLE
Feat: Explore, Profile, Bookmarks  page

### PR DESCRIPTION
**Pull request by sri-rishi**
### Added Explore, Profile, Bookmarks page and Added Routes

files to check

- Explore.jsx
- UserProfilePage.jsx
- Bookmarks.jsx
- Route.jsx
- Template.jsx

**Profile Page** 

image

**Explore page**

image

**Bookmark  page**

image

### CODE REVIEW

**Review by partha8 — APPROVED** [review]
Good clean code, and everything was readable. Good to merge

**Comment by varun-Tad** [comment]
Everything looks good. Also nice of you to have a separate file to keep all routes. never thought about it. You are good to merge.

File: src/pages/explore/Explore.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Post, SearchBox } from "../../components/index"
```
Comment (bajajcodes) [inline src/pages/explore/Explore.jsx:1]: use absolute imports

File: src/routes/TemplateRoute/Template.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    return (
+        <div>
+            <div className="w-full">
+            <Navbar />
+            <div className="md:flex w-full flex-row relative">
+                <MenuBar />
+                <main className="main w-full min-h-screen mt-16 pt-4 md:pt-0 flex flex-col items-center gap-4 border-x md:mt-0 md:w-5/6 md:ml-1/6">
+                    <Header />
+                    <div className="w-full flex flex-col items-center pb-36 md:pb-4 gap-4"> 
+                        <Outlet />
```
Comment (bajajcodes) [inline src/routes/TemplateRoute/Template.jsx:14]: Good use of outlet.

**Review by bajajcodes — COMMENTED** [review]
- Code is good and consistency is maintained.
- Code is simple and easy to understand, like made for humans to read, not only for machines.

**Pull request merged**